wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -1004,6 +1009,11 @@ impl PhysicalDeviceProperties {
             extensions.push(khr::shader_atomic_int64::NAME);
         }
 
+        // Require `VK_GOOGLE_display_timing` if the associated feature was requested
+        if requested_features.contains(wgt::Features::VULKAN_GOOGLE_DISPLAY_TIMING) {
+            extensions.push(ash::google::display_timing::NAME);
```

Comment:
This wasn't qualified above, `google` is imported in scope. Doesn't `wgpu` check for `unused_qualifications`?

```suggestion
            extensions.push(google::display_timing::NAME);
```